modules/ROOT/pages/storage.adoc

```diff
@@ -471,3 +535,4 @@ The only supported writable locations are `/etc` and `/var`. `/etc` should conta
 === Version selection and bootup
 
 A GRUB menu entry is created for each version of Fedora CoreOS currently available on a system. This menu entry references an `ostree` deployment which consist of a Linux kernel, an initramfs and a hash linking to an `ostree` commit (passed via the `ostree=` kernel argument). During bootup, `ostree` will read this kernel argument to determine which deployment to use as the root filesystem. Each update or change to the system (package installation, addition of kernel arguments) creates a new deployment. This enables rolling back to a previous deployment if the update causes problems.
+
```

Comment:
Minor: Seem unrelated?

Reply:
Seems so to me, but it was there before so I figured there must be some logic/history. Ask about retaining it or not at the meeting?

Reply:
Sorry, I meant the additional line added in this commit.